.vscode/settings.json

```diff
@@ -1,4 +1,9 @@
 {
 	"yaml.schemas": {},
-	"cSpell.words": ["aerospike", "kennylong", "kennylong's"]
```

Comment:
This looks like there is a war between formatters but please use the default trunk from this repository.